build.config

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
-CAKE_VERSION=0.38.5
-DOTNET_VERSION=3.1.402
+CAKE_VERSION=1.0.0-rc0001
+DOTNET_VERSION=5.0.100-rc.2.20479.15
```

Comment:
The build currently fails after the bumping of .NET version. See [here](https://dev.azure.com/cake-build/website/_build/results?buildId=13807&view=logs&j=fd490c07-0b22-5182-fac9-6d67fe1e939b&t=00933dce-c782-5c03-4a85-76379ccfa50a&l=87). Can we bump .NET version in a separate PR to not block this

Reply:
Fixed pipeline so it now builds.

Reply:
What have you changed? It also fails for me locally

Reply:
Are you running build.ps1 / build.sh?

Reply:
I adjusted so it executes build.ps1 instead of manually installing dotnet tool and executing it dotnet cake.

Reply:
@devlead Yes, running `.\build.ps1 --target=Preview` locally. First time it installed .NET but then failed with:

```
It was not possible to find any compatible framework version
```

Reply:
I installed .NET Runtime manually and now it works